pkg/kubectl/resource_printer.go

```diff
@@ -291,13 +293,25 @@ func (h *HumanReadablePrinter) unknown(data []byte, w io.Writer) error {
 	return err
 }
 
-func (h *HumanReadablePrinter) printHeader(columnNames []string, w io.Writer) error {
-	if _, err := fmt.Fprintf(w, "%s\n", strings.Join(columnNames, "\t")); err != nil {
+func (h *HumanReadablePrinter) printHeader(columnNames []string, withNamespace bool, w io.Writer) error {
+	columns := []string{}
+	if withNamespace {
```

Comment:
This approach is fine for now, but it's not scalable.  The next time someone comes in here, we're going to have to do something more dynamic (e.g. a map of column names to values)